### DISCUSSION TITLE
Setting an active cell

### DISCUSSION THREAD

**Discussion by don-odyssey** [PHPOffice/PhpSpreadsheet#3638] [Q&A] [answered]
Hi. I have a spreadsheet open for editing. First, I would like to set the active cell to start and then loop through the column until the last cell, identified by having no data. I have read through:

https://phpspreadsheet.readthedocs.io/en/latest/topics/accessing-cells/

but I only see ways to set a cell's value, creating a new cell, retrieving the cell value. There's no way to set the active cell???

I also note a feature to iterate through cells but that's not quite what I want (only for a column). However, there's no way to set the initial starting cell???

**Comment by MarkBaker** [comment, marked as answer]
The iterators do allow setting the starting cell for a row or for a column. The `getRowIterator()` and `getColumnIterator()` accept optional start and end arguments

**Comment by oleibman** [comment]
Does this work for you?
```php
$sheet->setSelectedCell('C3');
```
